src/core/absorption_chiller.jl

```diff
@@ -35,15 +35,15 @@
     chp_prime_mover::String = ""  # Informs thermal_consumption_hot_water_or_steam if not provided
 
     # Defaults for fields below are dependent on thermal_consumption_hot_water_or_steam and max cooling load
-    installed_cost_per_ton::Union{Float64, Nothing} = nothing
-    om_cost_per_ton::Union{Float64, Nothing} = nothing
-    min_ton::Float64 = 0.0,
-    max_ton::Float64 = BIG_NUMBER,
-    cop_thermal::Union{Float64, Nothing} = nothing,
-    cop_electric::Float64 = 14.1,
-    om_cost_per_ton::Union{Float64, Nothing} = nothing,
-    macrs_option_years::Float64 = 0,
-    macrs_bonus_fraction::Float64 = 0
+    installed_cost_per_ton::Union{Float64, Nothing} = nothing # Thermal power-based cost of absorption chiller (3.5 to 1 ton to kwt)
+    om_cost_per_ton::Union{Float64, Nothing} = nothing # Yearly fixed O&M cost on a thermal power (ton) basis
+    om_cost_per_kw::Float64 = NaN,  
```

Comment:
why do we have both per ton and per kw? looks like the per kw one just gets overwritten. in that case it shouldn't be a user input right?

Reply:
Agreed, it just gets overwritten based on the `om_cost_per_ton` input. I removed this from the user inputs in https://github.com/NREL/REopt.jl/pull/173/commits/8c36dbc025d18cb6604de27c06135fd7f1e10d63